Src/Screens/WhatsNewScreen/Viewables/PagerSlide.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  PagerSlide.swift
+//  project
+//
+//  Created by Gustaf Gunér on 2019-06-05.
+//
+
+import Foundation
+import Form
+import Flow
+import UIKit
+
+struct PagerSlide {
+    let title: String
+    let paragraph: String
+    let imageUrl: String
+}
+
+extension PagerSlide: Viewable {
+    func materialize(events _: ViewableEvents) -> (UIView, Disposable) {
+        let bag = DisposeBag()
+        
+        let view = UIView()
+        
+        let containerView = UIStackView()
+        containerView.alignment = .center
+        containerView.axis = .vertical
+        containerView.spacing = 8
+        containerView.isLayoutMarginsRelativeArrangement = true
+
+        let image = RemoteVectorImage()
+        image.pdfUrlStringSignal.value = imageUrl
+        
+        bag += containerView.addArranged(image) { imageView in
+            imageView.snp.makeConstraints { make in
+                make.width.centerX.equalToSuperview()
+                make.height.lessThanOrEqualTo(UIScreen.main.bounds.width > 320 ? 270 : 120)
```

Comment:
That won't work on a small device such as SE. The button will be pushed out of the screen due to this image taking up too much height. I don't think this will work very well on a very large device either, the image will be enormous compared to the text below 🤔

Reply:
I made a compromise by skipping explicitly stating the height and instead shrinking the width if the device is small